install.sh

```diff
@@ -50,9 +50,6 @@ else
     echo "LTSP server will be in Non-standalone mode of operation"	
     echo "There is an existing DHCP server running"
     echo "LTSP server won't provide DHCP services.."
-    sed -i /etc/dnsmasq.d/ltsp-server-dnsmasq.conf \
-        -e "/192.168.1.0,proxy\$/ c dhcp-range=${NETWORK}.0,proxy" \
-        -e "/10.0.2.0,proxy\$/ c dhcp-range=10.0.2.0,proxy"
 fi
```

Comment:
This is not what asked for. I was talking for the last line, you have removed the whole command.
You should understand what you are doing, not follow blindly my instructions (maybe I made a typing mistake).
And you have not tested it (at least not properly).
I am not satisfied with this way of working.

Reply:
Sorrry I was confused by what you meant with last line. 
Tests were sucessful as I was using default network address.
Will run another one with different network address.